lib/rubocop/cop/rspec/repeated_example.rb

```diff
@@ -40,7 +40,7 @@ def example_signature(example)
           key_parts = [example.metadata, example.implementation]
 
           if example.definition.method_name == :its
-            key_parts << example.definition.method_args
+            key_parts << [example.definition.first_argument]
```

Comment:
if this is a block type I think you could also destructure and use the `method_args` method I think it is still there on `SendNode` types